app/pulp/app/tasks/__init__.py

```diff
@@ -0,0 +1,2 @@
+from pulp.app.tasks import repository  # noqa
```

Comment:
Is there any particular reason to import task modules here and no the tasks themselves? No to put all tasks in the same namespace?